api/src/index.js

```diff
@@ -3,29 +3,77 @@ import express from "express";
 import cors from "cors";
 import bodyParser from "body-parser";
 import knex from "./database_client.js";
-import nestedRouter from "./routers/nested.js";
 
 const app = express();
 app.use(cors());
 app.use(bodyParser.json());
 
 const apiRouter = express.Router();
 
-// You can delete this route once you add your own routes
-apiRouter.get("/", async (req, res) => {
-  const SHOW_TABLES_QUERY =
-    process.env.DB_CLIENT === "pg"
-      ? "SELECT * FROM pg_catalog.pg_tables;"
-      : "SHOW TABLES;";
-  const tables = await knex.raw(SHOW_TABLES_QUERY);
-  res.json({ tables });
+app.get("/my-route", (req, res) => {
+  res.send("Hi friend");
 });
 
-// This nested router example can also be replaced with your own sub-router
-apiRouter.use("/nested", nestedRouter);
+// Respond with all meals in the future (relative to the 'when' datetime)
+app.get('/future-meals', async (req, res) => {
+  try {
+    const meals = await knex("meal").where("when", ">", knex.fn.now());
+    res.json(meals[0].length ? meals[0] : []); // meals[0] because knex.raw returns an array
```

Comment:
The requirement is to return all meals in the future
> /future-meals	Respond with all meals in the future (relative to the when datetime)

```suggestion
    res.json(meals);
```

Reply:
Thank you very much for the tips